index.js

```diff
@@ -58,7 +58,8 @@ def('Node',
     [$.Any, $Node],
     _node => ({
       '_@@type': 'sanctuary-html/Node',
-      equals: _other => $Node.test(_other) && String(_other) === String(_node),
+      equals: other => $Node.test(other) &&
+              _html(other.value) === _html(_node),
```

Comment:
Those two are equivalent, yes. However, the change above uses `_html(_node)` to determine equivalency.